extensions/package-vulnerability-scanner/src/components/ui/StatusMessage.vue

```diff
@@ -8,7 +8,7 @@ defineProps<{
 
 <template>
   <div
-    class="p-4 rounded-md border-l-4 mb-4"
+    class="p-4 rounded-r-md border-l-4 mb-4"
```

Comment:
This was done to match the vulnerabilities blocks in the list of vulnerabilities in a package.